build_library/sbsign_util.sh

```diff
@@ -3,44 +3,23 @@
 # found in the LICENSE file.
 
 if [[ ${COREOS_OFFICIAL:-0} -ne 1 ]]; then
-    SBSIGN_KEY="/usr/share/sb_keys/shim.key"
-    SBSIGN_CERT="/usr/share/sb_keys/shim.pem"
+    SBSIGN_KEY="/usr/share/sb_keys/unofficial/shim.key"
+    SBSIGN_CERT="/usr/share/sb_keys/unofficial/shim.pem"
 else
     SBSIGN_KEY="pkcs11:token=flatcar-sb-dev-hsm-sign-2025"
-    unset SBSIGN_CERT
+    SBSIGN_CERT="/usr/share/sb_keys/official/signing.pem"
```

Comment:
Since we can fetch it from Keyvault on demand, and this certificate is needed while signing, why store it in the repo. Better keep the way it's done.